bedrock/thunderbird/templates/thunderbird/start/release.html

```diff
@@ -31,8 +31,9 @@
       <div>
         <section class="help">
           <h2 class="iconic">{{ _('Need Support?') }}</h2>
+          {% set support_url = '/questions/new/thunderbird' if LANG in ['en-GB', 'en-US', 'en-ZA'] else '/kb/get-community-support' -%}
```

Comment:
Would it be better to check for locales that start with `en-` here as opposed to enumerate through the hard-coded list?